.github/workflows/continuous-integration.yaml

```diff
@@ -10,6 +10,42 @@ on:
       - master
 
 jobs:
+  fix-formatting:
+    if: >-
+      github.actor != 'github-actions[bot]' &&
+      github.event_name == 'pull_request' &&
+      github.repository == 'nock/nock'
```

Comment:
I'll have to figure out how to prevent running the fixer for forks, as we don't have permissions to push things to other repositories.

Reply:
`github.pull_request.head.repo.fork` is either true or false

Reply:
Great, thanks for the tip! Changed it.